user/deployment/pages.md

```diff
@@ -0,0 +1,48 @@
+---
+title: GitHub Pages Deployment
+layout: en
+permalink: /user/deployment/github-pages/
+---
+
+Travis CI can deploy your static files to [GitHub
+Pages](https://pages.github.com/) after a successful build.
+
+You will need to provide a [personal access
+token](https://help.github.com/articles/creating-an-access-token-for-command-line-use/)
+and set the deployment provider details in `.travis.yml`.
+
+For a minimal configuration, add the following to your `.travis.yml`:
+
+```yaml
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN # Set in travis-ci.org dashboard
```

Comment:
@webknjaz Good point, I missed the fact that this is in deploy.

@joepvd I'm not sure what you mean by the secrets not being exposed. Here is a sample repo and some sample builds that prove otherwise. You can see both the `SECRET` & `SECRET_DASHBOARD` variables being echoed in the `script`, `after_script`, and `after_success` keys:

* [Repo](https://github.com/lasley/test-travis-pr-vars)
* [Master Build](https://travis-ci.org/lasley/test-travis-pr-vars/builds/187510996)
* [Branch Build](https://travis-ci.org/lasley/test-travis-pr-vars/builds/187511229)
* [PR Build](https://travis-ci.org/lasley/test-travis-pr-vars/builds/187511938)

Obviously you can selectively expose the secrets like is done here, only showing it to deploy, but if the secret is exposed it is exposed from what I can tell.